intel_extension_for_transformers/llm/runtime/graph/scripts/ci/cpp_graph_inference.sh

```diff
@@ -0,0 +1,406 @@
+                    fi
+                    ## run inference
+                    export LANG=en_US.UTF-8
+                    export LC_ALL=en_US.UTF-8
+                    echo "=======  Inference Start  ======="
+
+                    real_ctx=$ctx # TODO(Zhenzhong): use same ctx for  chatglm & baichuan
+                    [[ "${model}" == "chatglm2" || "${model}" == "chatglm-6b" ||
+                        "${model}" == "baichuan-13b" || "${model}" == "baichuan2-13b" ]] && real_ctx=2047
+
+                    OMP_NUM_THREADS=$cores_per_instance numactl -m 0 -C 0-$(($cores_per_instance - 1)) \
+                        $infer_cmd --seed 1234 -t $cores_per_instance -b 2047 -c $real_ctx -n ${output} -m ${model}-${precision}.bin $extension -p "$prompt" 2>&1 | tee ${WORKSPACE}/${logs_file} || true &
+                    minitor
+
+                    echo "=======  Inference End  ======="
+                    python $script_dir/calculate_percentiles.py ${WORKSPACE}/${logs_file} ${model} ${precision} ${cores_per_instance} ${batch_size} ${input} ${output}
+
+                    if [[ "$cores_per_instance" == "${cores_list[@]: -1:1}" ]] &&
+                        [[ "$batch_size_idx" == "0" ]] &&
+                        [[ "$input_idx" == "0" ]] &&
+                        [[ "${#ppl_nctx_list[@]}" != "0" ]]; then
+                        ppl_eval "$task_name" "$cores_per_instance" "$model_path" "$model-$precision.bin"
+                    fi
+                done
+            done
+        done
+    done
+    if [[ "${#ppl_nctx_list[@]}" != "0" ]] && [[ "$ppl_fp32_test" = true ]]; then
+        cores_per_instance="${cores_list[@]: -1:1}"
+        task_name="${model}-fp32-${cores_per_instance}-${batch_size_list[@]:0:1}-${input_list[@]:0:1}-${output}"
+        ppl_eval "$task_name" "$cores_per_instance" "$model_path" "$model-fp32.bin"
+    fi
+    conda deactivate >/dev/null 2>&1
+}
+
+function minitor() {
+    sleep 1
+    # try first time
+    if [ $(ps -ef | grep "$infer_cmd" | wc -l) -lt 2 ]; then
+        #python $script_dir/calculate_percentiles.py ${logs_file} ${model} ${precision} ${cores_per_instance} ${batch_size} ${input} ${output}
+        sleep 1
+    fi
+    # keep monitoring
+    echo "======  Monitor Start ======="
+    while true; do
+        if [ $(ps -ef | grep "$infer_cmd" | wc -l) -lt 2 ]; then
+            #python $script_dir/calculate_percentiles.py ${logs_file} ${model} ${precision} ${cores_per_instance} ${batch_size} ${input} ${output}
+            sleep 3
+            break
+        fi
+        echo "$(date +%s), $(numastat -p $(ps -ef | grep "$infer_cmd" | grep -v grep | awk '{printf("%s  ", $2)}'))" >>${WORKSPACE}/memory.txt 2>&1
+    done
+    echo "======  Monitor End ======="
+}
+function get_data() {
+    python $script_dir/calculate_percentiles.py ${logs_file} ${model} ${precision} ${cores_per_instance} ${batch_size} ${input} ${output}
+}
+main $@ 2>&1 | tee ${WORKSPACE}/launch.log
```

Comment:
useless?

Reply:
`minitor` used in L361. (BTW, should be renamed to `monitor`).

`get_data` does not used.

Will be corrected soon.

Reply:
Updated in 41cfe88